app/routes/joblistings/components/JoblistingDetail.js

```diff
@@ -115,6 +120,30 @@ const JoblistingDetail = ({
               />
             </div>
           )}
+          {(canEdit || canDelete) && (
+            <ul>
+              <br />
+              <li>
+                <strong>Admin</strong>
+              </li>
+              {canEdit && (
+                <li>
+                  <Link to={`/joblistings/${joblisting.id}/edit`}>Rediger</Link>
+                </li>
+              )}
+              {canDelete && (
+                <ConfirmModalWithParent
+                  title="Slett jobbannonse"
+                  message="Er du sikker på at du vil slette denne jobbannonsen?"
+                  onConfirm={onDeleteJoblisting}
+                >
+                  <li>
+                    <Link to="">Slett</Link>
```

Comment:
don't need a `Link` here? if you want it to be red you can style that instead. the `li` should also probably be outside the confirm`ConfirmModalWithParent`?